appengine-java11/oauth2/pom.xml

```diff
@@ -0,0 +1,108 @@
+<?xml version="1.0" encoding="UTF-8"?><project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+  <modelVersion>4.0.0</modelVersion>
+  <packaging>war</packaging>
+  <version>1.0-SNAPSHOT</version>
+  <groupId>com.example.appengine</groupId>
+  <artifactId>oauth2-sample</artifactId>
+
+  <!-- <parent>
+    <groupId>com.google.cloud.samples</groupId>
+    <artifactId>shared-configuration</artifactId>
+    <version>1.0.10</version>
+  </parent> -->
+
+  <properties>
+    <maven.compiler.source>11</maven.compiler.source>
+    <maven.compiler.target>11</maven.compiler.target>
+  </properties>
+
+  <dependencies>
+    <!-- Compile/runtime dependencies -->
+    <dependency>
+      <groupId>com.google.appengine.demo</groupId>
+      <artifactId>simple-jetty-main</artifactId>
+      <version>1</version>
+      <scope>provided</scope>
+    </dependency>
+
+    <!-- OAuth 2.0 packages -->
+    <dependency>
+      <groupId>com.google.oauth-client</groupId>
+      <artifactId>google-oauth-client</artifactId>
+      <version>1.28.0</version>
+      <scope>provided</scope>
```

Comment:
Do these need to be scoped to provided?

Reply:
I believe so for the runtime.

Reply:
You may want to verify that these are provided by the runtime - Seems weird to me that they would be provided (I would think that would be a potential problem with versioning)

Reply:
The user needs to copy the dependencies into ${project.build.directory}/appengine-staging; that folder provides the dependencies to the runtime. This is what I understand from Ludo, but correct me if this is off.